### PR TITLE
[stable/elasticsearch] add support for extraContainers

### PR DESCRIPTION
**Pull request by Dariusch**
Signed-off-by: Dariusch Ochlast <dariusch.ochlast@gmail.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by cpanato** [comment]
@Dariusch i think in your commit there are other commits which may be not part of the elasticsearch. Can you please check your commits again?

Also please sign the DCO
thanks!

**Comment by Dariusch** [comment]
yeah well I tried to rebase on master but something went wrong.. i will have a look

**Comment by Dariusch** [comment]
done.

File: stable/elasticsearch/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: elasticsearch
 home: https://www.elastic.co/products/elasticsearch
-version: 1.18.0
+version: 1.18.1
```
Comment (cpanato) [inline stable/elasticsearch/Chart.yaml:3]: please bump to 
```suggestion
version: 1.19.0
```

adding new feature

**Comment by cpanato** [comment]
@Dariusch also please update your branch and resolve the conflict

**Pull request closed without merging**